source/common/upstream/original_dst_cluster.cc

```diff
@@ -194,10 +176,11 @@ OriginalDstClusterFactory::createClusterImpl(
     const envoy::api::v2::Cluster& cluster, ClusterFactoryContext& context,
     Server::Configuration::TransportSocketFactoryContext& socket_factory_context,
     Stats::ScopePtr&& stats_scope) {
-  if (cluster.lb_policy() != envoy::api::v2::Cluster::ORIGINAL_DST_LB) {
+  if (cluster.lb_policy() != envoy::api::v2::Cluster::ORIGINAL_DST_LB &&
```

Comment:
Thanks, this is awesome. Can you actually deprecate the ORIGINAL_DST_LB LB type enum, and put it in deprecated.rst? This will allow us to move the original_dst cluster to an extension in a future release which would be great.

Reply:
Done.